cachito/web/utils.py

```diff
@@ -0,0 +1,32 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+from flask import request, url_for
+
+
+def pagination_metadata(pagination_query):
+    """
+    Returns a dictionary containing metadata about the paginated query.
+    This must be run as part of a Flask request.
+    :param pagination_query: flask_sqlalchemy.Pagination object
+    :return: a dictionary containing metadata about the paginated query
+    """
+
+    pagination_data = {
```

Comment:
Can you set default values to `next` and `prev` so that they are just null when they aren't applicable.